install_config/install/advanced_install.adoc

```diff
@@ -681,6 +681,154 @@ openshift_examples_modify_imagestreams=true
 |===
 
 
+[[advanced-install-containerized-glusterfs-persistent-storage]]
+=== Configuring Containerized GlusterFS Persistent Storage
```

Comment:
@jarrpa I thought "Containerized" would be worth mentioning since we have two methods in general for configuring GlusterFS persistent storage ([CNS vs dedicated](https://docs.openshift.com/container-platform/3.5/install_config/persistent_storage/persistent_storage_glusterfs.html)). What do you think?